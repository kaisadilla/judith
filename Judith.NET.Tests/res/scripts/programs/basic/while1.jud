var i = 0

while i < 10 do
    print("i is now ")
    __p_print i

    i = i + 1
end