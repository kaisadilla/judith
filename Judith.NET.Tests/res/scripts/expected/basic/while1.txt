i is now 0
i is now 1
i is now 2
i is now 3
i is now 4
i is now 5
i is now 6
i is now 7
i is now 8
i is now 9
